index.html

```diff
@@ -367,13 +432,13 @@
                 </div>
             </a>
         </div>
-        <div id="chat"">
-                <div class="form-group height-full">
-                    <textarea class="form-control" id="chat-server" readonly></textarea>
-                </div>
-            <form role="form" onsubmit="server.chat(); return false;" class="height-full" autocomplete="off">
-                    <input type="text" class="form-control" id="chat-me"></textarea>
-                <button type="submit" class="btn btn-primary">Send</button>
+        <div id="chat">
+            <div class="form-group height-full">
+                <textarea class="form-control" id="chat-server" readonly></textarea>
+            </div>
+            <form role="form" onsubmit="server.chat(); return false;" class="height-auto" autocomplete="off">
+              <input type="text" class="form-control" id="chat-me">
+              <button id="send-button" type="submit" class="btn btn-primary">Send</button>
```

Comment:
Ordered the divs and removed redundant characters and strings.
Also added id to send button in order to be able to collapse it by script.